app/utils/pagination.py

```diff
@@ -29,3 +29,15 @@ def generate_previous_next_dict(view, service_id, page, title, url_args):
         "title": title,
         "label": "page {}".format(page),
     }
+
+
+def generate_pagination_pages(total_items, page_size, current_page):
+    total_pages = (total_items + page_size - 1) // page_size
```

Comment:
Was `// page_size` meant to be a comment?  If so, the `//` needs to be a `#` instead. 🙂 

```python
    total_pages = (total_items + page_size - 1) # page_size
```

Or were you trying to divide?  In which case, it looks like there's an extra `/` then:

```python
    total_pages = (total_items + page_size - 1) / page_size
```

Reply:
https://www.geeksforgeeks.org/floor-division-in-python/

I'm using floor division, which divides the first number by the second number and then rounds down to the nearest whole number.

Reply:
Totally missed that and forgot about it, thank you @heyitsmebev! 😀